quark/protocol-1.0.q

```diff
@@ -121,7 +317,7 @@ namespace mdk_protocol {
 
     class Open extends ProtocolEvent {
 
-        static Discriminator _discriminator = anyof(["open", "mdk.protocol.Open", "discovery.protocol.Open"]);
+        static Discriminator _discriminator = anyof(["open", "mdk_protocol.Open", "discovery.protocol.Open"]);
```

Comment:
The first value passed to anyof is used when encoding the Open event, the latter values are acceptable alternatives for decoding and there for the purpose of backwards compatibility, so changing to mdk.protocol to mdk_protocol probably doesn't make sense here. We should either remove it entirely because we don't care about compatibility with that particular version of the code, or we should leave it as mdk.protocol.